docs/howto/push-notifications.md

```diff
@@ -170,6 +170,64 @@ idiosyncratic capitalization shown.
 
 ### Current workaround
 
+This workaround means using a development server. If that won't work for you,
+you'll have to try the second workaround, below.
+
+We can tell our development server to talk to Apple's APNs "sandbox" server,
+instead of its server meant for production, but you'll need a certificate signed
+by Apple authorizing you to do so. Some background on that is
+[here](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns/#2947606).
+
+1. First, generate a Certificate Signing Request (CSR). Apple Developer's
+   instructions are
+   [here](https://help.apple.com/developer-account/#/devbfa00fef7). You can use
+   something like "Johnny Appleseed APNs Sandbox" for the "Common Name."
```

Comment:
Customarily "John Appleseed" in this context. (Comparison: [John](https://www.google.com/search?hl=en&q=%22john+appleseed%22+site%3Adeveloper.apple.com) vs. [Johnny](https://www.google.com/search?hl=en&q=%22johnny+appleseed%22+site%3Adeveloper.apple.com).)